hapi-proto/src/main/proto/ScheduleCreate.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2020 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+option java_multiple_files = true;
+
+import "BasicTypes.proto";
+import "Timestamp.proto";
+
+message ScheduleCreateTransactionBody {
+  bytes transactionBody = 1; // The transaction serialized into bytes that must be signed
+  Key adminKey = 2; // (optional) The Key which is able to delete the Scheduled Transaction (if tx is not already executed)
+  AccountID payerAccountID = 3; // (optional) The account which is going to pay for the execution of the Scheduled TX. If not populated, the scheduling account is charged
+  SignatureMap sigMap = 4; // The signature map containing the signature of at least one account defined in the signers proto on transaction creation
```

Comment:
If I recall, we originally planned to require a list of `AccountID`s to sign the wrapped txn; there seem echoes of this idea in the (perhaps deprecated) comment above,

> The signature map containing the signature of at least one account defined in the signers proto

I can't find such a list of required `AccountID` signers. 😀  Perhaps I've lost the thread, haha...

But if this signers list is indeed absent, I assume we should interpret the above `SignatureMap` as having two special properties:
1. Only full 32-byte Ed25519 public keys are allowed for the `SignaturePair#pubKeyPrefix` fields.
2. The `SignaturePair#ed25519` signature field _may_ be empty.

Seem correct?

Reply:
The list of `AccountID`s were discussed previously to be used as **define the list of accounts that can append signatures via ScheduleSign**. However, in our last discussion, we confirmed that we don't actually need them. 

The `SignatureMap` here actually holds the signatures for the scheduled transaction, rather than definition on who is able to execute `ScheduleSign`. In this case, if someone creates a Scheduled Transaction and along side the creation  of the scheduled transaction he is able to provide signatures for that scheduled transaction (similarly to executing ScheduledSign).

I think that the "special properties" you mention are not required in that case as we are not using the `sigMap` for defining "who should sign" but actually we are using the property for supplying signatures of the underlying scheduled transaction

Reply:
Updated the description of the `sigMap` to be more clear on its purpose